apis/service-providers/millom-standard-api.yaml

```diff
@@ -99,62 +234,51 @@ paths:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
         "504":
-          description: "Failure from upstream operator"
+          description: Failure from upstream operator
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+                
   /std/{serviceProvider}/{subscriberId}/purchasePlan:
     post:
-      summary: "Get upsellOffer"
-      description: "Retrieve a list of offers for a subscriber"
+      summary: Get upsellOffer
+      description: Retrieve a list of offers for a subscriber
       parameters:
```

Comment:
Should be "purchase Plan" both in summary and description, right ? :)

Reply:
yes, I have updted.